site/homepage/content/release-notes.md

```diff
@@ -10,6 +10,10 @@ description = "Information about changes in recent Hono releases. Includes new f
 
 * The CoAP adapter did not correctly track the time it took to forward a command message to a device. This has been fixed.
 
+### Deprecations
+The `org.eclipse.hono.client.ApplicationClientFactory`, `org.eclipse.hono.client.AsyncCommandClient` and
```

Comment:
please add an empty line and then add an `*` in order to turn this into an entry of an unnumbered list

Reply:
Thanks for noticing it and I have updated it now.